src/tree/xpath/XPath.ts

```diff
@@ -27,44 +28,38 @@ import { XPathWildcardElement } from "./XPathWildcardElement";
  * Represent a subset of XPath XML path syntax for use in identifying nodes in
  * parse trees.
  *
- * <p>
- * Split path into words and separators {@code /} and {@code //} via ANTLR
+ * Split path into words and separators `/` and `//` via ANTLR
  * itself then walk path elements from left to right. At each separator-word
- * pair, find set of nodes. Next stage uses those as work list.</p>
+ * pair, find set of nodes. Next stage uses those as work list.
  *
- * <p>
  * The basic interface is
- * {@link XPath#findAll ParseTree.findAll}{@code (tree, pathString, parser)}.
- * But that is just shorthand for:</p>
+ * {@link XPath#findAll ParseTree.findAll}`(tree, pathString, parser)`.
+ * But that is just shorthand for:
  *
- * <pre>
- * {@link XPath} p = new {@link XPath#XPath XPath}(parser, pathString);
- * return p.{@link #evaluate evaluate}(tree);
- * </pre>
+ * ```
+ * let p = new XPath(parser, pathString);
+ * return p.evaluate(tree);
+ * ```
  *
- * <p>
- * See {@code org.antlr.v4.test.TestXPath} for descriptions. In short, this
- * allows operators:</p>
+ * See `TestXPath` for descriptions. In short, this
+ * allows operators:
```

Comment:
Is there web-based documentation that explains the API.   This isn't covered in the latest ANTLR book, right?

Reply:
I'm not sure for this. I never really used the feature much.